django/applications/catmaid/apps.py

```diff
@@ -311,11 +310,11 @@ def validate_configuration(self) -> None:
             if not isinstance(getattr(settings, field), allowed_types): # type: ignore
                 current_type = type(getattr(settings, field))
                 if len(allowed_types) == 1:
-                    raise ImproperlyConfigured("Please make sure settings field %s "
-                            "is of type %s (current type: %s)" % (field, data_type, allowed_types[0]))
```

Comment:
I'm pretty sure this was a bug (last parameter was wrong)

Reply:
Agreed.